### PR TITLE
Switch to MIT license

### PR DESCRIPTION
**Pull request by jdhoffa**
Before merging this PR, we will need approval from all contributors

### CODE REVIEW

**Comment by maurolepore** [comment]
I approve the change to MIT license.

The other human authors are @jdhoffa and @2diiKlaus. Not sure how to get approval from the institutional author "2 Degrees Investing Initiative"; maybe you Klaus or Jakob can represent 2DII?



```
Authors@R: 
    c(person(given = "Mauro",
             family = "Lepore",
...
      person(given = "Jackson",
             family = "Hoffart",
...
      person(given = "Klaus",
             family = "Hagedorn",
...
      person(given = "2 Degrees Investing Initiative",
...
             email = "contact@2degrees-investing.org"))
```

File: LICENSE.md [hunk 1]
```diff
@@ -1,595 +1,21 @@
...
-information on this, and how to apply and follow the GNU GPL, see
-&lt;<http://www.gnu.org/licenses/>&gt;.
-
-The GNU General Public License does not permit incorporating your program into
-proprietary programs. If your program is a subroutine library, you may consider it
-more useful to permit linking proprietary applications with the library. If this is
-what you want to do, use the GNU Lesser General Public License instead of this
-License. But first, please read
-&lt;<http://www.gnu.org/philosophy/why-not-lgpl.html>&gt;.
+MIT License
```
Comment (maurolepore) [inline LICENSE.md:1]: I see this file but DESCRIPTION still mentions the old license. `usethis::use_mit_license()` should do the trick.

**Review by maurolepore — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 1 above)
Comment (maurolepore) [reply LICENSE.md]: Done.

**Review by maurolepore — APPROVED** [review]

**Comment by maurolepore** [comment]
@2diiKlaus, If you support the change to MIT license, can you please write a comment here stating so, e.g. 

> I approve the change to MIT license.

Also, who can approve on behalf of 2DII? Is it you too?

**Comment by maurolepore** [comment]
@jdhoffa, as the author of this PR I assume you support the change to MIT license; but to leave no doubts, can you please write a comment here stating so, e.g.

> I approve the change to MIT license.

**Comment by jdhoffa** [comment]
I approve the change to MIT license.

**Comment by 2diiKlaus** [comment]
I support changing the license of this package to MIT.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: 2 Degrees Investing Initiative France
```
Comment (maurolepore) [inline LICENSE:2]: I had expected this to read "2 Degrees Investing Initiative" -- without "France". 
Can you confirm?

**Review by maurolepore — CHANGES_REQUESTED** [review]
Did you use the usethis helper? I think a file is missing with the MIT license: LICENSE.md.

File: LICENSE (hunk 2 above)
Comment (jdhoffa) [reply LICENSE:2]: it should have France, it will be the French entity holding the license

**Comment by maurolepore** [comment]
@jdhoffa, given our discussion on Slack, it seems 2DII's directors approved this change. I suggest we merge and try get this to CRAN.

**Review by maurolepore — APPROVED** [review]
LGTM

**Pull request merged**